test/bdd/features/get-errors.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Get errors test
+  Background: Setup local blockchain, bootstraps and nodes
+    Given the blockchain is set up
+    And 1 bootstrap is running
+
+
+
+
+  @get-errors
+  Scenario: Getting non existent UAL
+    Given I setup 4 nodes
+    And I wait for 10 seconds
+    And I call get directly to ot-node 1 with nonExistentUAL
+    And I wait for last resolve to finalize
+    Then Last GET operation finished with status: GetAssertionIdError
+
+
+
+  @get-errors
+  Scenario: GET operation result on a node with minimum replication factor greater than the number of nodes
+    Given I setup 4 nodes
+    And I wait for 10 seconds
+    And I call publish on node 1 with validAssertion
```

Comment:
You can remove these steps (for successfull publish), since this test should only test get network error and not happy path for publish operation, we have dedicated test for that

Reply:
I think that if there are no published assertions, get operation will never reach the _NetworkGetCommand_ where _GET_NETWORK_ERROR_ is thrown, _GetAssertionIdError_ will be thrown before, as UAL we're trying to get is non-existent. So, i an assertion must be published before i try to get it.

Please, correct me if I'm wrong, so i can change the scenario.